test/tests/saga-saveFlattrs.js

```diff
@@ -0,0 +1,100 @@
+"use strict";
+
+const {assert} = require("../assert");
+const SagaTester = require("redux-saga-tester").default;
+const requireInject = require("require-inject");
+
+const {rootReducer} = requireInject("../../src/lib/background/state/reducers");
+const {
+  SAVE_FLATTRS,
+  SAVE_FLATTRS_SUCCESS
+} = require("../../src/lib/background/state/types/flattrs");
+const {spawn} = require("../utils");
+
+const TEST_PATH = "../../src/lib/background/state/sagas/saveFlattrs";
+
+const makeTestModule = () =>
+{
+  const deps = {
+    "global/window": {
+      setTimeout,
+      clearTimeout
+    },
+    "../../src/lib/background/database/flattrs/save": {
+      save({url})
+      {
+        return new Promise((resolve) =>
+        {
+          setTimeout(resolve, 0);
```

Comment:
Suggestion: I assume you'd like to use [`setImmediate()`](https://nodejs.org/api/timers.html#timers_setimmediate_callback_args) here instead.

Reply:
I'm not sure why I would want to `resolve` a promise multiple times.  It does look like I can get away with simply using `Promise.resolve()` here instead.